### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jango2106**
Added Steve

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,5 @@ Authors
 Dustin Roan (dustin.a.roan@gmail.com)
 
 Kevin Hoopes (kevin.alex.hoopes@gmail.com)
+
+STEVE!!!!! (Steve.steven@stevemail.xxx)
```
Comment (ahoopes16) [inline README.md:29]: But I hate Steve...

Comment (jango2106) [inline README.md:29]: Oh I added Steve because Steve you know

**Review by ahoopes16 — COMMENTED** [review]
Let's think twice about Steve, he wears Crocs...

File: README.md (hunk 1 above)
Comment (jango2106) [reply README.md:29]: Fair enough
NO STEVE!!!

**Pull request closed without merging**